state.go

```diff
@@ -83,6 +83,10 @@ func (v *View) Copy() *View {
 	return vv
 }
 
+func (v *View) String() string {
+	return fmt.Sprintf("(%d, %d)", v.Sequence, v.Round)
```

Comment:
Do you think it would be wise to add additional information to this print?
How would people know if a round or sequence come first, apart from diving into the code?

What do you think about making the print something like:
```go
return fmt.Sprintf("(Sequence [%d]; Round [%d])", v.Sequence, v.Round)
```